components/post_view/reaction/reaction.scss

```
@@ -66,6 +66,14 @@
         vertical-align: middle;
     }
 
+    &__emoji--post-menu {
+        min-width: 18px;
+        max-width: 18px;
+        min-height: 18px;
+        max-height: 18px;
```

nit: I guess we could simplify by just:
```suggestion
        width: 18px;
        height: 18px;
```